langchain/src/vectorstores/faiss.ts

```diff
@@ -254,6 +289,21 @@ export class FaissStore extends SaveableVectorStore {
     return instance;
   }
 
+  static async fromIndex(
```

Comment:
What's the purpose of this method? Wouldn't we just call `.load()`?

Reply:
Its not absolutely necessary but I think it provides a cleaner way to clone a previous index without saving on disk.

Reply:
One more point, It doesn't make sense to require initializing `fromDocuments` or `fromTexts` before they can merge. So `fromIndex` provides a cleaner way of doing: `const vectorStore = new FaissStore(new FakeEmbeddings(), {}); vectorStore.mergeFrom(...);`. Let me know what you think.